### PR TITLE
[Docs] Clarify background knowledge

### PR DESCRIPTION
**Pull request by ErikSchierboom**

### CODE REVIEW

File: docs/concept-exercises.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ When designing Concept Exercises, please consider the following guidelines:
 
 1. The exercise should be able to be solved in 5-10 minutes by someone proficient in the language.
 1. The exercise should not involve having to learn or engineer a non-trivial algorithm.
```
Comment (SaschaMann) [inline docs/concept-exercises.md]: I'm still unsure what a "trivial algorithm" is. Should that go in another PR or can it be discussed here?

Comment (ErikSchierboom) [reply docs/concept-exercises.md]: We can discuss it here.

Comment (ErikSchierboom) [reply docs/concept-exercises.md]: @SaschaMann I'll merge this and then we can submit a new PR later.
